### PR TITLE
fix(synthetic-shadow): composed copy, cut, and paste events

### PR DESCRIPTION
**Pull request by ekashida**
## Details

These clipboard events are not composed in Firefox
https://bugzilla.mozilla.org/show_bug.cgi?id=1620420

## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`

If yes, please describe the impact and migration path for existing applications.

## The PR fulfills these requirements:
* Have tests for the proposed changes been added? ✅ 
* Have you followed [these instructions](../CONTRIBUTING.md#-commit-message-conventions) to clearly describe the issue being fixed or feature enhanced? ✅

### CODE REVIEW

**Comment by pmdartus** [comment]
@ekashida Why did you removed the webdriver tests?

File: packages/@lwc/synthetic-shadow/src/polyfills/clipboard-event-composed/main.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
```
Comment (ravijayaramappa) [inline packages/@lwc/synthetic-shadow/src/polyfills/clipboard-event-composed/main.ts]: Import the common object methods from `@lwc/shared`

**Comment by ekashida** [comment]
@pmdartus I was encountering similar issues as last time...javascript errors happening on the browser that weren't available through the sauce reporting, and something is really wrong with my sauce connect proxy setup since I'm not able to successfully run remote tests on any browser other than Chrome.

File: packages/integration-tests/src/components/events/test-clipboard-event-composed/clipboard-event-composed.spec.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+            const didHandleCopy = browser.execute(() => {
+                var container = document.querySelector('integration-clipboard-event-composed');
+                return container.didHandleCopy();
+            });
+
+            assert.strictEqual(didHandleCopy, true);
+        });
+
+        // The cut() function is not working as expected.
```
Comment (pmdartus) [inline packages/integration-tests/src/components/events/test-clipboard-event-composed/clipboard-event-composed.spec.js]: Can you add more details on why the cut method doesn't work as expected? Is it true for all browsers?

Comment (ekashida) [reply packages/integration-tests/src/components/events/test-clipboard-event-composed/clipboard-event-composed.spec.js]: I have no idea why it doesn't work. We should have sufficient coverage via `copy` and `paste` so I removed it.

**Pull request merged**